source/material_model/interface.cc

```diff
@@ -836,6 +836,17 @@ namespace aspect
 
 
 
+    template <int dim>
+    NamedAdditionalMaterialOutputs<dim>::
+    NamedAdditionalMaterialOutputs(const std::vector<std::string> &output_names,
+                                   const unsigned int n_points)
+      :
+      names(output_names),
+      output_values(output_names.size(), std::vector<double>(n_points, numbers::signaling_nan<double>()))
```

Comment:
You need to switch these two lines so that the order is the same as in the header file, I think this is causing the warning.

Reply:
Thanks for the quick reply :) ... fingers crossed!